### PR TITLE
style: add font-weight on lyric

### PR DESCRIPTION
**Pull request by mikelxk**
The current lyric is just too small in any perspecitve.

### CODE REVIEW

**Comment by Dumeng** [comment]
Currently, most of CSS in the single file. It's too hard to modify.
We need to extract them into separated views and components.

Also, the flex layout is much better than margin and padding.

And we also may need to utilize some modern CSS tooling framework such as TailwindCSS to simply the codes.

**Comment by mikelxk** [comment]
Can't agree more!!!
I tried to use tailwind earlier but found out it's just too much work.
I will try to make a pr to convert css to tailwind manually or by scripts(hopefully) in the next month.

**Comment by listen1** [comment]
> Can't agree more!!! I tried to use tailwind earlier but found out it's just too much work. I will try to make a pr to convert css to tailwind manually or by scripts(hopefully) in the next month.

good idea. I think it would make it easier to write css.

about this pr, i think we'd better make lyric font size as option in setting for users, because users may have different requirement for that.

**Comment by listen1** [comment]
one more thing about new css framework migration: theme switching maybe a hard problem to solve. In origin code, we inject theme css file to html to solve this problem, I'm not sure how css framework handle that problem in vue environment.

**Comment by mikelxk** [comment]
> one more thing about new css framework migration: theme switching maybe a hard problem to solve. In origin code, we inject theme css file to html to solve this problem, I'm not sure how css framework handle that problem in vue environment.

I see a blog https://www.samdawson.dev/article/tailwind-theming on that, that might be an easy approach.

**Comment by mikelxk** [comment]
I've created corresponding fields in setting, but I didn't implement the user toggleable button because we're still developing here.

File: src/composition/settings.ts [hunk 1]
```diff
@@ -28,7 +28,10 @@ const settings: Record<string, unknown> = reactive({
   enableGlobalShortCut: false,
   enableNowplayingPlatform: false,
   enableLyricTranslation: false,
-  theme: 'black'
+  theme: 'black',
+  //lyric settings
+  lyricFontSize: 20,
```
Comment (listen1) [inline src/composition/settings.ts:33]: lyricFontSize: 20,
  lyricFontWeight: 500,

considering user habit, please set default value as origin version

  lyricFontSize: 15,
  lyricFontWeight: 400,

**Pull request merged**